internal/cmd/cmd.go

```diff
@@ -90,11 +89,9 @@ func apiClient(host string, accessKey string, skipTLSVerify bool) (*api.Client,
 
 	u.Scheme = "https"
 
-	headers := http.Header{}
-	ua := fmt.Sprintf("Infra CLI/%v (%v/%v)", internal.Version, runtime.GOOS, runtime.GOARCH)
-	headers.Add("User-Agent", ua)
-
 	return &api.Client{
+		Name:      "cmd",
```

Comment:
`CLI` or `cli` seems like a more obvious name than `cmd` for our CLI user agent. This name doesn't need to match the package, right?

Reply:
You're probably right. I was going for consistency between the client name and their package, e.g. `internal/connector` is `connector` and `internal/cmd` is `cmd`